packages/shared/isArray.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+declare function isArray(a: mixed): boolean %checks(Array.isArray(a));
+
+const isArrayImpl = Array.isArray;
+
+// eslint-disable-next-line no-redeclare
+function isArray(a: mixed): boolean {
+  return isArrayImpl(a);
```

Comment:
Why can't you just export Array.isArray? Is this to work around a Flow issue?

Reply:
Yes

Reply:
Or rather, to work around my lack of knowledge if there is a way in Flow to express this function declaration without an function.